### ISSUE TITLE
Ligatures Not Appearing In Hyper

### ISSUE THREAD

**Issue by Mario253** [tonsky/FiraCode#779]
I recently switched from gnome-terminal to hyper, and decided to give this a try. I had tried previously on gnome-terminal before I saw it was unsupported (see #778), so I thought this would work better. Instead of some characters not appearing, no ligatures worked at all. Here's a screenshot of some failed ligatures:
![Screenshot 2019-06-09 at 08 23 57](https://user-images.githubusercontent.com/32368758/59159498-0082b300-8a90-11e9-957f-38fe52cc9324.png)

**Comment by tonsky** [comment]
This one? https://hyper.is/ Linux?

**Comment by Mario253** [comment]
Yes, that is the correct link. I am running it in Crostini, so that could have something to do with it.
